openlibrary/templates/account/loans.html

```diff
@@ -55,6 +55,9 @@
 <div>
   $if len(loans) == 0:
     <div><em>$_("You've not checked out any books at this moment.")</em></div>
+    $:render_template("books/custom_carousel", books=readonline_carousel(), title=_('Classic Books'), url="/read", key="public_domain", test=test, compact_mode=True)
+    $:render_template("home/categories", test=test)
+    Check out our <a href="/collections">special collections</a>
```

Comment:
```suggestion
    Or, check out our <a href="/collections">special collections</a>.
```
Adding "Or" to the beginning of this sentence makes more sense with the call to action.